### PR TITLE
Fix sharing input placeholder for emails

### PR DESCRIPTION
**Pull request by skjnldsv**
Because we can always get a user by its email

### CODE REVIEW

**Review by MorrisJobke — APPROVED** [review]
👍

File: apps/files_sharing/src/components/SharingInput.vue [hunk 1]
```diff
@@ -117,22 +117,16 @@ export default {
...
 			}
-			if (allowRemoteSharing && !allowMailSharing) {
-				return t('files_sharing', 'Name or federated cloud ID …')
-			}
-			if (allowRemoteSharing && allowMailSharing) {
-				return t('files_sharing', 'Name, federated cloud ID or email address …')
-			}
 
-			return t('files_sharing', 'Name …')
+			return t('files_sharing', 'Name, federated cloud ID or email address …')
```
Comment (jancborchardt) [inline apps/files_sharing/src/components/SharingInput.vue:129]: Just a detail, but let’s shorten it a bit and make sure the most common parts are sorted first:
> Name, email, or federated cloud ID …

Same above:
> Name or email …

Comment (skjnldsv) [reply apps/files_sharing/src/components/SharingInput.vue]: Done

**Review by jancborchardt — APPROVED** [review]

**Pull request merged**